pwndbg/gdblib/regs.py

```diff
@@ -246,6 +255,9 @@ def _fs_gs_helper(self, regname: str, which: int) -> int:
         """Supports fetching based on segmented addressing, a la fs:[0x30].
         Requires ptrace'ing the child directory if i386."""
 
+        if not register_exists(regname):
```

Comment:
Hmm I think this should be only executed for x86-64 case, so within the if that is in line 261.

Reply:
This is probably why the tests on CI fail after this change 😅